### PR TITLE
Add note to capture all the improvements to client server connection

### PR DESCRIPTION
**Pull request by lmbr-pip**
Add line about improvements to the client/server connection experience.

Example of new client to server connection step messaging:
multiplayer_connection_text

Example of new connection status text in game:
connection_status

Signed-off-by: Pip Potter <61438964+lmbr-pip@users.noreply.github.com>

### CODE REVIEW

File: releases/22.10.0/22100 feature list.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The general theme of this release is "TBD".
 
 
 **sig-network**
+* Multiple improvements to the client server connection experience, including debug text to convery current step in connection process, along with connection status ImGui display and new ImGUI menu options.
```
Comment (AMZN-Gene) [inline releases/22.10.0/22100 feature list.md:37]: “Multiple improvements to the client and server connection experience, including debug text in the viewport to convery current step in connection process, connection status, and new ImGUI menu options.”
- Changed “client server” to “client and server” just to be sure it’s not confused with the client server host type.
- Connection status is displayed as debug text in the viewport separate from the ImGui menu (although it was suggested adding it there as well)

**Review by AMZN-Gene — APPROVED** [review]

File: releases/22.10.0/22100 feature list.md (hunk 1 above)
Comment (lmbr-pip) [inline releases/22.10.0/22100 feature list.md:37]:
```suggestion
* Multiple improvements to the client and server connection experience, including debug text to convey the current step in connection process, debug connection status messaging in game and new ImGUI menu options in the Launchers.
```

Comment (lmbr-pip) [reply releases/22.10.0/22100 feature list.md]: See new edit for update

**Review by tonybalandiuk — COMMENTED** [review]
@lmbr-pip Can you link to docs, a PR, or some other place where the reader can learn more? Are there any screenshots or examples of this debug text or status messages?

**Comment by lmbr-pip** [comment]
I can capture some screenshots, let me know if there are any requirements here in terms of size

**Comment by tonybalandiuk** [comment]
@lmbr-pip If it's easier to link to docs or other than that's probably easiest. As for images here, we haven't really done that before, so I don't have guidance on sizes. If you want to go with screenshots here in this file, how about you try a couple and we'll go from there.

**Comment by lmbr-pip** [comment]
@tonybalandiuk Images now attached to PR. Let me know if theres anything else you want here. Unsure where they should go in reference to release notes.

**Comment by tonybalandiuk** [comment]
@lmbr-pip  I made a change to your file to add a link to this PR so readers can find the images.

**Pull request merged**